spec/models/curator/shared/filestreams/derivative_methods.rb

```diff
@@ -12,18 +12,11 @@
     end
   end
 
-  describe '#derivatives_payload' do
-    it { is_expected.to respond_to(:derivatives_payload) }
+  describe '#ark_params' do
```

Comment:
Should this be `avi_params`? Also overall this spec is a lot less detailed than the one for `derivatives_payload` -- suggest either fleshing it out or adding a TODO if the payload is still somewhat of a work in progress?

Reply:
oh that was a typo on my part. I was going to skip it for now until we have fixtures for all the file sets that require derivatives and when the avi processor is operational